.github/actions/composite/configureAwsCredentials/action.yml

```diff
@@ -18,4 +22,4 @@ runs:
       with:
         aws-access-key-id: ${{ inputs.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ inputs.AWS_SECRET_ACCESS_KEY }}
-        aws-region: us-east-1
+        aws-region: ${{ inputs.AWS_REGION }}
```

Comment:
I see what you mean about this action being pointless now ... annoying that local composite actions from the same repo don't have access to secrets.